includes/Includes_Neopets_Auctions/README.md

```diff
@@ -0,0 +1,17 @@
+
+# [Includes : Neopets : Auctions](.)
```

Comment:
README.md isn't required because I execute a script that creates/updates all README.md automatically.

If you want to add some explanation on how the file works, add it to README-APPEND.md
It should be appended to README.md by the same script mentioned before.